apim-operator/pkg/controller/api/api_controller.go

```diff
@@ -595,19 +658,28 @@ func getCredentials(r *ReconcileAPI, name string) error {
 	//get the username and the password
 	for k, v := range credentialSecret.Data {
 		if strings.EqualFold(k, "username") {
-			basicUsername = string(v)
+			usrname = string(v)
 		}
 		if strings.EqualFold(k, "password") {
-			//encode password to sha1
-			_, err := hasher.Write([]byte(v))
+			password = v
+		}
+
```

Comment:
Shall we remove the unnecessary new lines?

Reply:
This is already fixed in the latest code